Cloud9 Support for Stacksets (#227)
aws-samples/cfn101-workshop: content/intermediate/operations/stacksets/index.md

    :::code{language=shell showLineNumbers=false showCopyAction=true}
    aws cloudformation delete-stack-set \
    --stack-set-name cfn-workshop-ec2instance 
    :::
    1. Follow the steps 1-2 for other two stack setrs, and in the following order: `cfn-workshop-security`, and `cfn-workshop-network`.
    ::::
    ::::tab{id="local" label="Local development"}    
    
    **How to delete AWS CloudFormation stacks within stack set**
    
    1. Navigate to the [AWS CloudFormation StackSets console.](https://console.aws.amazon.com/cloudformation/home#/stacksets)
    1. Select the CloudFormation stack set you want to delete the stacks from. Choose the last stack set you created, i.e., `cfn-workshop-ec2instance`.
    1. From top-right section of the page, select **Actions**, and choose **Delete stacks from StackSet**.
    1. Under **Accounts**, select **Deploy stacks in accounts** under **Deployment locations**.
    1. Under **Account numbers** enter the 12-digit AWS account ID for the account you are using for this lab.
    1. For **Specify regions** select **Add all regions**. This will automatically select the AWS Regions that the StackSet deployed stacks into. Choose **Next**.

>     1. The **Status** changes to `PENDING`.

Reviewer: ```suggestion
    1. On the **Review** page, review the contents of the page, and choose **Submit**.
    1. The **Status** changes to `PENDING`.
```

    1. Refresh until the **Status** changes to `SUCCEEDED`.
    1. Follow steps 2 through 8 for the other two stack sets, and in the following order: `cfn-workshop-security`, and `cfn-workshop-network`.

    Now that you have deleted stacks within each StackSet, you will now choose to delete the empty StackSet.
    
    **How to delete an AWS CloudFormation stack set**
    
    
    1. Navigate to [AWS CloudFormation StackSets console](https://console.aws.amazon.com/cloudformation/home#/stacksets).

>     2. Select the stack set you wish to delete.

Reviewer: ```suggestion
    1. Select the stack set you wish to delete.
```

>     3. Choose **Actions**, and then **Delete StackSet**.

Reviewer: ```suggestion
    1. Choose **Actions**, and then **Delete StackSet**.
```

>     4. Under **Accounts**, select **Deploy stacks in accounts** under **Deployment locations**.

Reviewer: ```suggestion
```

>     5. In the popup that appears, confirm you want to delete this stack set by choosing **Delete StackSet**.

Reviewer: ```suggestion
    1. In the popup that appears, confirm you want to delete this stack set by choosing **Delete**.
```

>     6. On refresh, your StackSet should no longer be listed.

Reviewer: ```suggestion
    1. On refresh, your StackSet should no longer be listed.
```

>     7. Follow steps 2 through 6 for the two other stack sets.

Reviewer: ```suggestion
    1. Follow steps 2 through 5 for the two other stack sets.
```
